README.md

```diff
@@ -1,3 +1,273 @@
+
+    > Do not use file `acre.yaml` in folder `crop`. This file is used internally by the installation tool.
+
+1. If you created the base cluster using GKE you need to convert your `KUBECONFIG` to one that uses basic authentication with Google-specific configuration parameters:
+
+    1. Set your `KUBECONFIG` environment variable to a file that does not exist. Do *not* use the one specified in your acre.yaml file.
+
+    1. Download the GKE kubeconfig:
+
+        ```bash
+        gcloud container clusters get-credentials <your_cluster> --zone <your_zone> --project <your_project>
+        ```
+
+    1. Call `crop/bin/convertkubeconfig 3.1`. When asked for credentials, enter the ones that the GKE dashboard shows when clicking on `show credentials`.
+
+    This will create a kubeconfig that uses basic auth. It will be created at the path specified in your `acre.yaml` file.
+
+1. Open a second terminal window and enter the following command to watch the progress of the Gardener installation:
+
+    ```bash
+    watch -d kubectl -n garden get pods,ingress,sts,svc
+    ```
+
+1. In your first terminal window, use the following command to deploy Gardener on your base cluster:
+
+    ```bash
+    sow deploy -A
+    ```
+
+`sow` now starts to install Gardener in your base cluster. The installation can take about 30 minutes. `sow` prints out status messages to the terminal window so that you can check the state of the installation. The other terminal window will show the newly created Kubernetes resources after a while and if their deployment was successful (their status should be `Running`). Finally, the Gardener dashboard is deployed. The last status message before `sow` exits is something like this:
+```bash
+===================================================================
+Dashboard URL -> https://gardener.ingress.<value of `landscape.cluster.domain`>
+===================================================================
+generating exports
+*** species dashboard deployed
+```  
+
+More information: [Most Important Commands and Directories](#most-important-commands-and-directories)
+
+## Configuration File acre.yaml
+
+This file will be evaluated using `spiff`, a dynamic templating language for yaml files. For example, this simplifies the specification of field values that are used multiple times in the yaml file. For more information, see the [spiff repository](https://github.com/mandelsoft/spiff/blob/master/README.md).
+
+<pre>
+landscape:
+  <a href="#landscapename">name</a>: &lt;URL-friendly name&gt;                       # Gardener managed k8s landscape name
+
+  <a href="#landscapecluster">cluster</a>:                                          # Information about your base cluster
+    kubeconfig: &lt;relative path + filename&gt;          # Path to your `kubeconfig` file, rel. to folder `landscape`
```

Comment:
Since the docker container that runs `sow` only mounts parts of the surrounding file system, absolute paths are actually not supported here. To avoid problems, the kubeconfig should be within the landscape directory (or a subdirectory of it).

Reply:
Not sure what you mean here. It says `relative path` above, so all should be fine, right?